specs/capri/object_lifecycle_management.md

```diff
@@ -0,0 +1,525 @@
+
```

Comment:
Add a title to the document.

Add authors like this:
**Author(s)**:  [Shruthi MN](https://github.com/Shruthi-1MN), [Pravin Ranjan](https://github.com/PravinRanjan10)
Replace with your name and Neelam.

Reply:
Ok, all comments about format will be accepted.